content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'scaleOrdinal()'
+Description: 'scaleOrdinal() function and examples'
```

Comment:
make this description a small explanation of what scaleOrdinal() does and then explain it more below this metadata.

Reply:
Alright